README.md

```diff
@@ -136,3 +136,28 @@ If you do not have Python 3.10 installed, you can use root-less podman with a Py
 ```bash
 podman run --rm --tty --volume "$(pwd):/mnt:z" --workdir /mnt docker.io/library/python:3.10 bash -c 'pip install nox ; nox -s pip-compile'
 ```
+
+## Creating release tags
+Creating tags that correspond with releases and release candidates in [`ansible/ansible`](https://github.com/ansible/ansible)
+
+First, install the additional tagging dependencies
+
+```bash
+pip install gitpython typer
+```
+
+Next, you'll need to ensure that you have both [`ansible/ansible`](https://github.com/ansible/ansible) and [`ansible/ansible-documentation`](https://github.com/ansible/ansible-documentation) checked out in sibling directories.
+
+Lastly, run the various tasks with the tagging tool.
+
+Show tags from ansible-core that are not present here:
+```bash
+./hacking/tagger/tag.py new-tags
+```
+
+Determine missing `anisble-core` releases from `--branch`. Create corresponding tags for releases in `ansible-documentation`, and push them:
+```bash
+./backing/tagger/tag.py tag
+```
```

Comment:
You might also consider adding an example with the `--remote` option because not everyone (including me) sets the upstream as origin. Maybe combine into a single codeblock like this:

```bash
# The tagger script assumes origin as the upstream remote.
./hacking/tagger/tag.py tag

# If you use something else as the upstream remote, specify the name.
./hacking/tagger/tag.py tag --remote <name> 
```